### ISSUE TITLE
Lein Create Statements (HW3)

### ISSUE THREAD

**Issue by amit70** [csula/cs590-fall-2015#16]
Can you provide those 3 lein create statements for HW3

**Comment by willy1087** [comment]
Adding to what @amit70 is asking. Could you also create a simple working example on how those work? I know in class you show us some path modification but it seems you went a little faster when explaining it. A working example will help a lot since this will show us how to combine 2 different folders (client and server).

**Comment by dnparmar** [comment]
Yeah it would be very helpful if you could post about that !!

**Comment by john-csula** [comment]
ok the idea here is to create two separate projects:

``` bash
mkdir Homework3
cd Homework3
lein new gradesheet-cljs
lein new luminus gradesheet
```

Then update the `gradesheet-cljs/project.clj` with the example [project.clj](https://github.com/csula/cs590-fall-2015/blob/master/src/event-pattern/project.clj).  

Finally in your new `project.clj` change `:output-to "resources/public/js/client.js"` so that it points to `../resources/public/js/client.js`.  The idea here is to show that you can generate `client.js` in an area where the files would be served up.

**Comment by willy1087** [comment]
@john-csula I was wondering, when I am trying this right now I can't see the client.js being created. I just want to clarify things. When you mention update `"gradesheet-cljs-/project.clj"`, I pretty much delete its content and replace it with what is in project.clj. Then I changed the path of output-to to what was given. However looking into the path, I changed it to this path instead (since I want this file to be created inside gradesheet `"../gradesheet/resources/public/js/client.js"` . when I run `lein cljsbuild once`, it looks (based on the timestamp) like it's trying to create it but at the end it doesn't. Anything that I am missing?
